app/Jobs/CreateOrUpdateUserFromBuzzAPI.php

```diff
@@ -136,6 +147,13 @@ public function handle(): void
         if ($user->is_service_account) {
             throw new Exception('BuzzAPI job attempted to create/update an account for an existing service account');
         }
+
+        foreach (self::EXPECTED_ATTRIBUTES as $attr) {
+            if (! property_exists($account, $attr)) {
+                throw new Exception('Selected account for '.$searchValue.' is missing expected attribute '.$attr);
+            }
+        }
```

Comment:
This doesn't seem to work and I'm not sure why. The same code works in the import job.